src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs

```diff
@@ -107,44 +108,51 @@ the previous head state.*/
                 preWarmTask = null;
                 WaitForCacheClear();
                 Block suggestedBlock = suggestedBlocks[i];
-                if (blocksCount > 64 && i % 8 == 0)
-                {
-                    if (_logger.IsInfo) _logger.Info($"Processing part of a long blocks branch {i}/{blocksCount}. Block: {suggestedBlock}");
```

Comment:
This log was helpful in determining that something went seriously wrong in the past :D

Reply:
Was just moved down